src/rev_ai/apiclient.py

```diff
@@ -159,6 +159,24 @@ def get_transcript_object(self, id_):
 
         return Transcript.from_json(response.json())
 
+    def delete_job(self, id_):
+        """Delete a specific transcription job
+        All data related to the job, such as input media and transcript, will be permanently
+        deleted. A job can only by deleted once it's completed.
+
+        :param id_: id of job to be deleted
+        :returns: True if job was successfully deleted
+        :raises: HTTPError
+        """
+        if not id_:
+            raise ValueError('id_ must be provided')
+
+        url = urljoin(self.base_url, 'jobs/{}'.format(id_))
+        response = self.session.delete(url)
+        response.raise_for_status()
+
+        return True
```

Comment:
why not just `return`? It seems that this can never return anything other than true - if delete fails, it will throw - so there doesn't seem to be a reason to return boolean.